modules/api/pkg/handler/common/machine.go

```diff
@@ -775,6 +779,18 @@ func outputMachine(machine *clusterv1alpha1.Machine, node *corev1.Node, hideInit
 	}, nil
 }
 
+func gpuLabels(labels map[string]string) map[string]string {
+	nvidiaLabels := make(map[string]string)
+
+	for k, v := range labels {
+		if strings.Contains(k, "nvidia.com") {
```

Comment:
Please create a constant for `nvidia.com`. 
Also, what if there is some other GPU than Nvidia? Or do we only want to display Nvidia labels?

Reply:
Thanks @Waseem826 for pointing this out! I've updated the PR and created a constant for nvidia.

For your second question, yes, at the moment, only NVIDIA is under consideration.